test/unit/HeartbeatTest.js

```diff
@@ -0,0 +1,36 @@
+// Third-party dependencies
```

Comment:
Since the `convertStateArrayToObject` function is in the `index.js` file, it would be great if this could be put into a `indexTest` folder. And then the name of the file will be the name of the function that you're testing: `convertStateArrayToObjectTest.js`.

So this will end up being `test/unit/indexTest/convertStateArrayToObjectTest.js` so that it will be easy to find.